snips_nlu_metrics/metrics.py

```diff
@@ -40,6 +42,8 @@ class must inherit from `Engine`
             and `actual_slot` corresponds to the slot as returned by the NLU
         progression_handler (lambda, optional): handler called at each
             progression (%) step
+        num_workers: number of workers to use. Each worker is assigned a
+            certain number of splits
```

Comment:
Can you add the type: `num_workers (int, optional)` and specify `(default_value=1)` in the description ?